theme.json

```diff
@@ -278,18 +343,18 @@
 					"customFontSize": false
 				},
 				"color": {
-					"custom": false,
-					"background": false
+					"custom": false
 				}
 			}
 		},
 		"custom": {
 			"line-height": {
-				"tiny": 1.125,
-				"small": 1.266,
-				"baseline": 1.602,
-				"medium": 1.802,
-				"large": 2.027
+				"baseline": 1,
```

Comment:
@Vinsanity We use the "baseline" line-height variable on line 393, making it the standard line-height for all elements inside the body tag that don't have an explicit line-height declared (in CSS or theme.json). Since this PR changes that value from 1.6 to 1, should we change line 393 to use to the xx-large value? If we don't, every element without an explicit line-height value defaults to a line-height of 1.

Reply:
If we leave it as is, here is the typography I see on my sample site. https://share.cleanshot.com/XtDTCcmKYk6fD170mMbs/

Reply:
That's a good discovery, Rob. After investigating the other designs, 1.5/150% is the common line-height used on standard text so I'll change that to use xx-large.

Reply:
@knice This is updated.